### ISSUE TITLE
CMake complains about no runtime destination

### ISSUE THREAD

**Issue by krishean** [k4zmu2a/SpaceCadetPinball#110]
https://github.com/k4zmu2a/SpaceCadetPinball/blob/de13d4e326b2dfa8e6dfb59815c0a8b9657f942d/CMakeLists.txt#L222

When I try to compile with CMake 3.10.2 on Ubuntu 18.04 gives the error:

```
CMake Error at CMakeLists.txt:222 (install):
  install TARGETS given no RUNTIME DESTINATION for executable target
  "SpaceCadetPinball".
```

I think this can be fixed by adding ` DESTINATION bin` to line 222 like so:

`	install(TARGETS "${PROJECT_NAME}" RUNTIME DESTINATION bin)`

Note: CMake on Ubuntu 20.04 and MSYS2/MinGW doesn't seem to have any issue with it being left off, it's only CMake on Ubuntu 18.04 that complains.

**Comment by NicolaSmaniotto** [comment]
This is strange, from the [CMake documentation](https://cmake.org/cmake/help/latest/command/install.html):
> For regular executables, static libraries and shared libraries, the DESTINATION argument is not required.
Probably 18.04 is still using an old version of CMake (<3.14) that didn't support this.
Can you try with `DESTINATION "${CMAKE_INSTALL_BINDIR}"`?

**Comment by krishean** [comment]
```
    include(GNUInstallDirs)
    install(TARGETS "${PROJECT_NAME}" RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}")
```

That works, without the include line it still fails with the same error message.

**Comment by NicolaSmaniotto** [comment]
Great. You should open a PR, or just wait and I'll do it in the next days.

**Comment by krishean** [comment]
I can wait a bit, there's no rush. Thanks for your help!

**Comment by k4zmu2a** [comment]
Moving this along, to be included in the upcoming release.

**Comment by k4zmu2a** [comment]
The fix for this issue was released in v2.0.1
